### DISCUSSION TITLE
does the V2.0.7 rans with a Chip ESP32-D0WD-V3 (revision v3.0) Board?

### DISCUSSION THREAD

**Discussion by kendo55** [espressif/arduino-esp32#7974] [Q&A]
does the V2.0.7 rans with a Chip ESP32-D0WD-V3 (revision v3.0) Board?

With old V1.0.5 everything is o.k.
With V2.0.7 i have severals Guru Meditation Error: Core  1 panic'ed (LoadProhibited)

**Comment by mrengineer7777** [comment]
Those are vastly different branches.  You'll need to update your project to work with the 2.0.x version.

**Comment by Jason2866** [comment]
yes, ESP32-D0WD-V3 does run fine with all Arduino cores.
